layouts/partials/head.html

```diff
@@ -19,10 +19,9 @@
 
   <!-- Bootstrap CSS -->
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
-  <!-- Portal CSS -->
-  <link rel="stylesheet" href="https://pathogens.se/css/styles.css">
```

Comment:
Just wondering, could we read the stylesheet from the portal's github repo directly?

Reply:
Yes, thought about it, but I personally felt it wouldn't look that nice :)